sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprActorsIT.java

```diff
@@ -53,7 +53,7 @@ public class DaprActorsIT {
   private static final Random RANDOM = new Random();
   private static final int PORT = RANDOM.nextInt(1000) + 8000;
 
-  private static final String ACTORS_MESSAGE_PATTERN = ".*Actor API level in the cluster has been updated to 10.*";
+  private static final String ACTORS_MESSAGE_PATTERN = ".*Found Subscription.*";
```

Comment:
@mcruzdev I agree for now.. but this is getting very inconsistent.. I keep finding edge cases..

Reply:
What edge cases @salaboy? I tried here n times and the tests are passing.

Reply:
@mcruzdev the problem is that if there are no subscriptions registered by the application this test way of waiting will not work. But for this, if you are creating a subscription for sure its ok, but it is not generic enough for all the test.

Reply:
I agree with you, for this case we really need to wait the subscription (see liine 91). But we can not use this approach as generic as well.